Automattic/wp-calypso
# Fix "All domains list" header style (WIP)

## Changes proposed in this Pull Request

This PR addresses some style fixes for the new header in All domains list:

* Height should be 70px, currently it’s 72px with some extra margin on top which makes it look not centered.
* It should be “sticky” when scrolling.
* The color of “?” next to the title should be var(–studio-gray-30).
* Filter dropdown: In single site view “All my domains” isn’t centered.
* Mobile: height should be 60px on mobile (*Update from Nov. 29*)
* Mobile: “3 dots” menu should also be 40 x 40px, with no border.
* Mobile: margin between the header and the filter dropdown should be 16px, currently it’s bigger.

pcYYhz-pX-p2#comment-356
pcYYhz-pX-p2#comment-354

## Screenshot

![header-desktop](https://user-images.githubusercontent.com/2797601/143909607-8a20e141-eaff-4061-a498-42b6cf755671.png)

![header-mobile](https://user-images.githubusercontent.com/2797601/143909614-e80a836b-5dbb-4282-b9a6-8c203c5f6e98.png)

## Testing instructions

* Build this branch locally or open the live Calypso link
* Go to "Upgrades > Domains" _(header it's same for All sites/Single site)_
* If you're in the live Calypso link, please append ?flags=domains/management-list-redesign to your URL to enable the appropriate feature flag
* Verify that the style of the header meets all the feedbacks above (use screenshots as reference)
